web-registry/src/components/organisms/DescriptionForm/DescriptionForm.constants.ts

```diff
@@ -0,0 +1 @@
+export const DESCRIPTION_MAX_LENGTH = 1400;
```

Comment:
@blushi I think that 1400 is probably fine if we have a "read more" link implemented on the front-end, which it looks to me like we don't have anymore (or maybe we only ever had it for the "additional story" section?). Otherwise maybe we should limit it to more like 1,000 characters.

Reply:
Ideally we would have the read more though

Reply:
@erikalogie looking at the code, we currently have the "read more" only for the land steward story but not for the generic description, I can create an issue to handle that

Reply:
i can't recall a reason that i set it to 1400, i probably was just copying from something that already existed in the standards. just wanted to mention that

Reply:
https://github.com/regen-network/regen-registry/issues/1569